app/models/export.rb

```diff
@@ -37,6 +37,14 @@ def to_following_accounts_csv
     end
   end
 
+  def to_follower_accounts_csv
+    CSV.generate(headers: ['Account address'], write_headers: true) do |csv|
+      account.passive_relationships.includes(:account).reorder(id: :desc).each do |follower|
```

Comment:
You need to do find_each here, you could be dealing with over 300,000 rows here.

Reply:
Thanks for the quick feedback, I'll switch this to the `.find_each` batch interface.